spec/services/merge_requests/create_service_spec.rb

```diff
@@ -5,21 +5,30 @@
   let(:user) { create(:user) }
 
   describe :execute do
-    context "valid params" do
+    context 'valid params' do
+      let(:opts) do
+        {
+            title: 'Awesome merge_request',
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.